packages/suite/src/views/dashboard/components/AssetsView/components/AssetRow.tsx

```diff
@@ -110,13 +111,18 @@ const CryptoBalanceWrapper = styled(Col)`
     }
 `;
 
-const FiatBalanceWrapper = styled.div`
-    color: ${({ theme }) => theme.textSubdued};
-`;
+const FiatBalanceWrapper = styled.div``;
 
 const ExchangeRateWrapper = styled(Col)`
     font-variant-numeric: tabular-nums;
     padding-right: 0;
```

Comment:
remove this, it's also why validation is failing because you're later setting padding-right again

Reply:
:heavy_check_mark: